atlasdb-backup/src/main/java/com/palantir/atlasdb/backup/AtlasBackupService.java

```diff
@@ -67,20 +88,25 @@ public static AtlasBackupService create(
     }
 
     private void storeBackupToken(InProgressBackupToken backupToken) {
-        storedTokens.put(backupToken.getNamespace(), backupToken);
+        inProgressBackups.put(backupToken.getNamespace(), backupToken);
     }
 
-    // TODO(gs): actually persist the token using a persister passed into this class.
-    //   Then we have an atlas-side implementation of the persister that conforms with the current backup story
+    // TODO(gs): validate backup ID?
     public Set<Namespace> completeBackup(Set<Namespace> namespaces) {
         Set<InProgressBackupToken> tokens = namespaces.stream()
-                .map(storedTokens::remove)
+                .map(inProgressBackups::remove)
                 .filter(Objects::nonNull)
                 .collect(Collectors.toSet());
         CompleteBackupRequest request = CompleteBackupRequest.of(tokens);
         CompleteBackupResponse response = atlasBackupClientBlocking.completeBackup(authHeader, request);
 
         return response.getSuccessfulBackups().stream()
+                // Store coordination service state locally.
```

Comment:
The code makes sense, but this comment is confusing -- we don't actually need to talk to timelock at all to get and store the coordination service state